src/components/button/examples/button-link/index.njk

```diff
@@ -2,6 +2,7 @@
 {{
     onsButton({
         "type": 'button',
+        "submitType": "timer",
```

Comment:
Do we need to add this for this example? Or did you just add this for testing?

Reply:
Just for something to test. Can remove when you have tested